docs/docs/how-tos/manual-backup.md

```diff
@@ -176,6 +176,34 @@ cd /data
 gsutil cp gs://<your_bucket_name>/backups/2021-04-23.tar .
 ```
 
+### Azure
+
+To do a backup on Azure, first install [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli-linux?pivots=script#install-or-update-azure-cli) and [azcopy](https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azcopy-v10?tabs=dnf#obtain-a-static-download-link). Otherwise, the instructions are the same as for AWS above, other than when working with S3. Here are the commands to access Azure blob storage instead of S3 for copy/download of the backup:
+
+#### Do the backup
+
+```bash
+az login # --use-device-code if web browser not available
+
+cd /data
+# Tell AZCOPY to use same auth as CLI
+export AZCOPY_AUTO_LOGIN_TYPE=AZCLI
+# copy the tar backup file to blob storage
+azcopy copy 2021-04-23.tazcopy cp "https://[account].blob.core.windows.net/[container]/2021-04-23.tar" "./2021-04-23.tar"ar 'https://<storage-account>.blob.core.windows.net/<blob-bucket>/backups/nebari-2021-04-23.tar'
```

Comment:
```suggestion
azcopy copy 2021-04-23.tazcopy cp "https://[account].blob.core.windows.net/[container]/2021-04-23.tar" "./2021-04-23.tar" 'https://<storage-account>.blob.core.windows.net/<blob-bucket>/backups/nebari-2021-04-23.tar'
```

I think that's a stray `ar` ?

Reply:
I'm also not familiary with these tools, but the command looks suspicious. Is it correct?